lib/index.js

```diff
@@ -253,7 +261,7 @@ class Goblin {
               );
             }
 
-            if (!msg.data._goblinCaller) {
+            if (!msg.data._goblinCaller && !msg.data._goblinCallerQuest) {
```

Comment:
I think that it should be:
```suggestion
            if (!msg.data._goblinCaller || !msg.data._goblinCallerQuest) {
```
otherwise the case where `msg.data._goblinCallerQuest` is true but not `msg.data._goblinCaller` will continue and it should not because `msg.data._goblinCaller` is always mandatory.